GetDKAN/dkan
# Dataset collections et

Connects https://github.com/isdapps/adc-code/issues/1674
Connects https://github.com/isdapps/adc-code/issues/1679
Connects https://github.com/isdapps/adc-code/issues/1680
Connects https://github.com/isdapps/adc-code/issues/1681
Connects https://github.com/isdapps/adc-code/issues/1682

## QA
[ ] Add a dataset to a parent via the collection field on the node form
[ ] After saving node, confirm collection link to parent appears in the metadata table
[ ] Visit the parent, and confirm the collection header reads "In this Collection"
[ ] Confirm there is a see all link
[ ] Confirm there is a total number of results displayed
[ ] Confirm body (description) field of parent is above collections block (you may need to edit dataset to add text).
[ ] Visit the search page, and confirm the Collection facet reads "Collection"
[ ] Confirm Collection facet is closed (by default)
[ ] Up to reviewer: task called for researching limit in facet to prevent user from getting overwhelmed with dozens of collections. Facets by default has a max limit of 20, but I set it to 15. If you want to populate multiple parents to test, cool, but I think we are safe.h
[ ] Add a new dataset.
[ ] Confirm collections field appears on the Additional data page underneath author field.
[ ] Confirm there is help text link underneath the collections field.
[ ] Edit an existing dataset, and add a node to the collection field.
[ ] Edit another existing dataset, that is not the parent item you added to the previous dataset, and verify that the previous dataset is not available in the collections autocomplete field.
